src/main/target/common_pre.h

```diff
@@ -335,6 +335,7 @@
 #define USE_RX_RSNR
 #define USE_RX_LINK_QUALITY_INFO
 #define USE_RX_MSP_OVERRIDE
+#define USE_RX_MSP_OVERWRITE
```

Comment:
:thinking: as the code is isolated we should not add it by default. This should be a custom define in the [cloud] build.

Reply:
At least we need some documentation for this [awesome] functionality

Reply:
I agree with you, documentation is needed.

also need to add information about activating this flight mode to OSD

Since everyone always forgets in "the field" - which switch is assigned which mode but on the table you remember everything 😀

Reply:
> At least we need some documentation for this [awesome] functionality

How can I add documentation to this resource [wiki](https://betaflight.com/docs/wiki) ?

Reply:
Can help with that. Need to add a page to the wiki archive or development section

Reply:
Like to have some documentation on this. There is a README in the project.